test/test_integration.py

```diff
@@ -41,7 +41,7 @@
                  'script': "shortname: extra_newline\ntable: extra_newline, http://example.com/extra_newline.txt",
                  'expect_out': 'col1,col2,col3\n1,2,3\n'}
 
-tests = [simple_csv, autopk_csv, crosstab, autopk_crosstab, skip_csv, extra_newline]
+tests = [simple_csv, autopk_csv, crosstab, autopk_crosstab, skip_csv]
```

Comment:
Is the test giving failing when it shouldn't or did the new changes trigger a change in behavior? Either way I'd prefer to get this fixed as part of the PR rather than turning off the test.